core/src/tower_storage.rs

```diff
@@ -94,7 +109,10 @@ impl SavedTower {
             )));
         }
 
-        let data = bincode::serialize(tower)?;
+        // SavedTower always stores its data in 1_14_11 format
```

Comment:
Is the idea here that the latency bit is only needed on chain so it's fine to just stick to the original tower format locally?

Reply:
Yes, the tower doesn't care about vote credits and it doesn't need to track latency.  In my opinion having the tower use the vote account's data structures is problematic as it conflates the use of those data structures between two very different subsystems.  It makes the tower dependent on the vote program, which is awkward.  The change I have made divorces the on-disk data format of the tower from that of the vote program, which I think is a step forward in addressing this.

Reply:
yeah I agree, tower barely uses anything in vote_state apart from the Vec<Lockout>, will be a pita but would be nice to separate.

Reply:
I have added a test in the newest version of this change, to verify that the expected vote account state is written under various circumstances.